### ISSUE TITLE
[Question/Enhancement] Is it possible to add headshot mechanics for modded entities?

### ISSUE THREAD

**Issue by LilJagty** [MrCrayfish/MrCrayfishGunMod#405]
This mod pairs well with the Target Dummies mod to make a gun range with shown damage values, but they show no indication of being headshot and the damage count stays the same no matter where the dummy is shot. 

How would it be possible to add headshot capabilities to the target dummies? Is it possible within a datapack or config, or would that require a PR for this mod or Target Dummy?

**Comment by MrCrayfish** [comment]
Yes, but only via code. I'll keep this open since I like the idea using a datapack to configure headshot hitboxes.
